modules/library/main/src/main/java/org/geotools/renderer/crs/WrappingProjectionHandler.java

```diff
@@ -240,6 +258,24 @@ public Geometry postProcess(MathTransform mt, Geometry geometry) {
         }
     }
 
+    /**
+     * In some cases the geometry is preflipped due to coordinate order and proximity to the
+     * dateline
```

Comment:
Write an example, 3 points polygon or 2 points line that would show what pre-flipped means.

Reply:
added